### PR TITLE
Add support for DoctrineOpenTransactionLoggerMiddleware

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

**Comment by lyrixx** [comment]
I'm not sure for the base branch

**Comment by stof** [comment]
shouldn't the old abstract service also be kept for BC ?

**Comment by stof** [comment]
to me, this is a new feature, so targetting 2.6.x makes sense.

**Comment by lyrixx** [comment]
Oups, I messed'up the patch, I'll clean it! (fixed)

File: Resources/config/messenger.xml [hunk 1]
```diff
@@ -28,6 +28,16 @@
             <argument type="service" id="doctrine" />
         </service>
 
+        <!--
+        The following service isn't prefixed by the "doctrine.orm" namespace in order for end-users to just use
+        the "doctrine_open_transaction_logger" shortcut in message buses middleware config
+        -->
+        <service id="messenger.middleware.doctrine_open_transaction_logger" class="Symfony\Bridge\Doctrine\Messenger\DoctrineOpenTransactionLoggerMiddleware" abstract="true" public="false">
+            <argument type="service" id="doctrine" />
+            <argument type="constant">null</argument>
```
Comment (stof) [inline Resources/config/messenger.xml]: `type="constant"` with `null` as name looks suspicious to me

Comment (ostrolucky) [reply Resources/config/messenger.xml]: Indeed. Even if it works, this is not conceptually correct. Pls reomve the type @lyrixx

Comment (lyrixx) [reply Resources/config/messenger.xml]: I'm not sure to understand. What should I use?

Comment (ostrolucky) [reply Resources/config/messenger.xml]: `<argument>null</argument>`

Comment (lyrixx) [reply Resources/config/messenger.xml]: Thanks. fixed.

**Review by ostrolucky — APPROVED** [review]

**Pull request merged**